index.html

```diff
@@ -575,12 +576,65 @@
      * @param {function} _t - A reference to function: i18next.t
      */
     function onLanguageSet(err, _t) {
-        if (err) 
+        
+        if (err)
             return console.log('Error while initializing locales: ', err);
         
         console.log('Language:', i18next.language);
         console.log(_t('key'));
-        console.log(_t('Aztecs'));
+
+        var elements = document.querySelectorAll('[data-i18n]');
+        for (var i=0, len=elements.length; i<len; ++i) {
+
+            var domElem = elements[i];
+            var locKey = domElem.getAttribute('data-i18n');
+
+            if (!locKey) {
+                console.warn('Could not localize: (' + locKey + ')');
+                continue;
+            }
+
+            // Set localized string back into the DOM element's end-user visible property
+            switch (domElem.nodeName) {
+
+                case 'OPTION':
+                    domElem.innerText = _t(locKey);
+                    break;
+                    
+            }
+        }
+    }
+
+
+    /**
+     * Iterates over all DOM elments that need to be localized, and 
+     * sets them up for the localization library to interact with.
+     *
+     * The text in English is what defines the keys for other all other
+     * languages.
+     */
+    function prepareDomForLocalization() {
```

Comment:
Instead of using this function, can't we just directly type the values in the `<option>` tags? The chance of the civilisation names changing frequently is very low.

Reply:
That is correct.  I chose not to go that route only to avoid text-duplication, but if you prefer that approach I can update the code and probably remove this function altogether.